src/reducers/signupReducer.js

```diff
@@ -0,0 +1,22 @@
+// @call type going to be used..
+import { SET_SIGNUP_FORM_INPUT, USER_ERRORS } from '../actions/type';
+
+const initialState = {};
+const reducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case SET_SIGNUP_FORM_INPUT:
+      return {
+        ...state,
+        signupForm: payload
```

Comment:
give this key a better variable

Reply:
Sorry, I am not sure I understand well what you mean here. | More clarity needed|

Reply:
`signupForm` is not the best key for this payload. You can simply rename it to `data`